posts/2020-08-05-jakarta-grpc-beta-20009.adoc

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: "More Jakarta EE 9 features and gRPC support in Open Liberty 20.0.0.9 beta"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/lauracowen
+author_github: https://github.com/lauracowen
+seo-title: More Jakarta EE 9 features and gRPC support in Open Liberty 20.0.0.9 beta - OpenLiberty.io
+seo-description: Try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools, either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try.
+blog_description: Try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools, either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try.
+---
+= More Jakarta EE 9 features and gRPC support in Open Liberty 20.0.0.9 beta
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+:url-prefix:
+:url-about: /about/
+
+We now have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
+* <<allbeta, All Beta Features>>: a new, larger package that contains all our non-Jakarta EE 9 beta features, any other Open Liberty features that are necessary for them to run, _and_ the Jakarta EE 9 features.
+
+This means that you can now try out our in-development Open Liberty features by just adding the relevant coordinates to your build tools (either the Jakarta EE 9 Beta Features package or the All Beta Features package, depending on what you want to try).
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their namechange completed. We've also made good TCK progress with these features.
```

Comment:
Is "namechange" one word or two?  Maybe it depends on which side of the pond you are on... ;-)

Reply:
I would also say "excellent TCK progress" instead of "good TCK progress".  Or, if that's too optimistic for you, how about "solid TCK progress"?  Good just sounds too average and I think we're doing a fine job with TCK testing.

Reply:
- "namechange" - just a typo, now fixed. :) 
- TCK progress is now "excellent" instead of mediocre. Even as a Brit I can gather a bit of optimism... ;)